### ISSUE TITLE
Time selection macros in query parameters

### ISSUE THREAD

**Issue by Vamic** [marcusolsson/grafana-json-datasource#17]
Many data sources have the $__timeFrom() and $__timeTo() macros, is it possible to add those?

I need the format in epoch for my integration but i imagine there would be many APIs that use different formats so maybe $__timeFrom("format") would be appropriate?

**Comment by marcusolsson** [comment]
Most of the built-in data sources also support `$__unixEpochFrom()`, `$__unixEpochTo()`. Since it covers your use case, and since many people are already used to them, maybe we can start there?

I'm thinking maybe we could add it to the queries even, to be able to filter even if the API doesn't support it. Something like:

```js
$.items[?(@.created_at > $__unixEpochFrom() && @.created_at < $__unixEpochFrom())].name
```

**Comment by marcusolsson** [comment]
@Vamic Would #22 solve your use case?

**Comment by Vamic** [comment]
Yep, that should be perfect. 👍

**Comment by marcusolsson** [comment]
Available now in [v0.7.0](https://grafana.com/grafana/plugins/marcusolsson-json-datasource?version=0.7.0)! 🎉
